.tekton/main-pipeline.yaml

```diff
@@ -465,6 +465,10 @@ spec:
     - input: $(params.build-source-image)
       operator: in
       values: [ "true" ]
+    # Pushing source container for main is known to fail a lot with timeout in Quay. See KFLUXBUGS-1508
+    # By trying a few more times, we hope to make this step succeed and not fail the pipeline.
+    # TODO: remove retries once KFLUXBUGS-1508 gets resolved.
```

Comment:
I did not do `TODO(KFLUXBUGS-1508)` because the ticket does not belong to us and the functioning of `TODO(<ticket-id>)` is very specific - <https://github.com/stackrox/stackrox/blob/master/scripts/ci/jobs/check-pr-fixes.sh>. There's hardly any chance we'll open a PR with a title starting with `KFLUXBUGS-1508:` in this stackrox repo.

I could have created `ROX-` issue but I'm afraid it will just die in the backlog.

I would go ahead and merge without following the suggestion. If you think it's worthwhile, please let me know and I'll follow up with a new PR.

Reply:
Resolving to be able to merge.